utils/fetch-retry-and-confirm-version.ts

```diff
@@ -51,10 +48,7 @@ async function fetchRetry(args: FetchRetryArgs) {
   const data = await r.text()
 
   if (JSON.stringify(data) !== JSON.stringify(expectedData)) {
-    console.log('expected:')
-    console.log(JSON.stringify(expectedData))
-    console.log('but got:')
-    console.log(JSON.stringify(data))
+    console.log(diff(expectedData, data))
```

Comment:
This is amazing, but let's still output the full strings as they are returned anyway to make it easier to copy/paste them to e.g. replace the "snapshot". (Maybe the diff first, and then the two explicit ones?)